src/core/utils/components/VirtualScroll.test.tsx

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import VirtualScroll from './VirtualScroll';
+
+const component = (
+  <div style={{ overflow: 'auto', height: 400 }} id='scroller'>
+    <VirtualScroll
+      itemsLength={1000}
+      itemRenderer={(index) => (
+        <div key={index} className='element' style={{ height: 40 }}>{`Element${
+          index + 1
+        }`}</div>
+      )}
+    />
+  </div>
+);
+
+const renderComponent = () => {
+  return render(component);
+};
+
+const renderEmptyComponent = () => {
+  return render(
+    <div style={{ overflow: 'auto', height: 400 }} id='scroller'>
+      <VirtualScroll
+        itemsLength={1000}
+        itemRenderer={(index) => <div key={index} className='element' />}
+      />
+    </div>,
+  );
+};
+
+beforeAll(() => {
+  // return correct values for container 'scroller' and children
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function getBoundingClientRectMock(this: Record<string, any>) {
```

Comment:
Why not inline in `mockImplementation`?
Also you could use `<span>` as a child, that way you could mock `HTMLDivElement.getBoundingClientRect` and `HTMLSpanElement.getBoundingClientRect` but I think it is ok how it is done now.

Reply:
changed, but left divs